locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -105,6 +105,8 @@ COPY package.json .
 # COPY package.json package-lock.json ./
 
 RUN npm install
+# If you are building your code for production
+# RUN npm install --only=production
```

Comment:
Isn't it better to use the `NODE_ENV` env var?

Reply:
Good point but an environment variable affects the following commands.
But I'm ok for changing it using `ARG NODE_ENV=production`

Reply:
It can be set when running `docker run` as per [best practices](https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#docker-run)

Reply:
Yes but `npm install` is made during the build time. The "-e" option is available only at run time.

If you prefer I can suggest to build the image using `--build-args NODE_ENV=production` in the best practise

Reply:
Yes you are right, I have no strong opinions.